src/ontology/edits/oeo-social.omn

```diff
@@ -199,6 +199,8 @@ Class: OEO_00000045
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A producer is an agent that makes goods.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/314
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/370",
```

Comment:
how do you do that? how can you link an issue to your changes? I think the pull request link is created automatically? these ar no real comments - just questions for my understanding

Reply:
we decided to use the "term tracker item" annotation to add links to pull request and issue. They are created by hand, I copy the links from github into the annotation.
https://github.com/OpenEnergyPlatform/ontology/wiki/term-tracker-item

Reply:
thanks for explaining